MCprep_addon/materials/default_materials.py

```diff
@@ -0,0 +1,216 @@
+# ##### BEGIN GPL LICENSE BLOCK #####
```

Comment:
Is there a default blend file that you have locally? You should include this in your commit too - and add it using git lfs add <filename> so it is trcaked as an object file and not tracked for diffs like code.